tests/cli/test_execute.py

```diff
@@ -28,6 +28,9 @@ def test_execute_cloud_flow_fails():
     result = runner.invoke(execute, "cloud-flow")
     assert result.exit_code == 1
     assert "Not currently executing a flow within a cloud context." in result.output
+    assert "Not currently executing a flow within a cloud context." in str(
```

Comment:
Is it possible to capitalize `Cloud` here (and in the actual error)?

Reply:
Yea sure!

Reply:
updated